### PR TITLE
Broker : Updating to Interop 0.13.2 and passing Broker Options Header text to Interop

### PR DESCRIPTION
**Pull request by gladjohn**
Fixes #
[Internal tracking item VS#1988070)](https://identitydivision.visualstudio.com/Engineering/_workitems/edit/1988070)

**Changes proposed in this request**
- Updated to Microsoft.Identity.Client.NativeInterop 0.13.2, that fixes a bug around ORG workaround. ESTS fixed the issue and MSAL Runtime removed the local fix. This fixes an issue with getting AAD account token in MSA-PT scenario
- Providing Broker options header text to Interop that will add a title to the AAD Account picker (works only on non-admin mode). Bug [2159876](https://identitydivision.visualstudio.com/Engineering/_workitems/edit/2159876), is tracking a fix for admin mode header title

**Testing**
Dev App

**Performance impact**
None

**Documentation**
- [ ] All relevant documentation is updated.

### CODE REVIEW

File: src/client/Microsoft.Identity.Client.Broker/WamAdapters.cs [hunk 1]
```diff
@@ -150,11 +151,11 @@ private enum ResponseStatus
         /// Gets the Common Auth Parameters to be passed to Native Interop
         /// </summary>
         /// <param name="authenticationRequestParameters"></param>
-        /// <param name="isMsaPassthrough"></param>
+        /// <param name="brokerOptions"></param>
         /// <param name="logger"></param>
         public static NativeInterop.AuthParameters GetCommonAuthParameters(
             AuthenticationRequestParameters authenticationRequestParameters, 
-            bool isMsaPassthrough,
+            WindowsBrokerOptions brokerOptions,
```
Comment (SameerK-MSFT) [inline src/client/Microsoft.Identity.Client.Broker/WamAdapters.cs:158]: is this a breaking change?

File: src/client/Microsoft.Identity.Client.Broker/RuntimeBroker.cs [hunk 2]
```diff
@@ -123,7 +123,7 @@ private static void OnProcessExit(object sender, EventArgs e)
             {
                 using (var authParams = WamAdapters.GetCommonAuthParameters(
                     authenticationRequestParameters, 
-                    _wamOptions.MsaPassthrough, 
+                    _wamOptions,
```
Comment (SameerK-MSFT) [inline src/client/Microsoft.Identity.Client.Broker/RuntimeBroker.cs:126]: Does this affect any runtime test(s)?

File: src/client/Microsoft.Identity.Client.Broker/WamAdapters.cs (hunk 1 above)
Comment (gladjohn) [reply src/client/Microsoft.Identity.Client.Broker/WamAdapters.cs:158]: It shouldn't be, earlier we were only using WindowsBrokerOptions for MSA-PT in `GetCommonAuthParameters`, now that we are also passing HeaderTitle to Interop, modified the method to accept `WindowsBrokerOptions` instead.

File: src/client/Microsoft.Identity.Client.Broker/RuntimeBroker.cs (hunk 2 above)
Comment (gladjohn) [reply src/client/Microsoft.Identity.Client.Broker/RuntimeBroker.cs:126]: no

File: src/client/Microsoft.Identity.Client.Broker/WamAdapters.cs [hunk 3]
```diff
@@ -172,9 +173,13 @@ private enum ResponseStatus
             authParams.RedirectUri = authenticationRequestParameters.RedirectUri.ToString();
 
             //MSA-PT
-            if (isMsaPassthrough)
+            if (brokerOptions.MsaPassthrough)
                 authParams.Properties[NativeInteropMsalRequestType] = ConsumersPassthroughRequest;
 
+            //WAM Header Title
+            if (!string.IsNullOrEmpty(brokerOptions.HeaderText))
```
Comment (bgavrilMS) [inline src/client/Microsoft.Identity.Client.Broker/WamAdapters.cs:180]: Assuming you have tested silent flow and this is still ok.

**Review by bgavrilMS — APPROVED** [review]

File: src/client/Microsoft.Identity.Client.Broker/WamAdapters.cs (hunk 3 above)
Comment (pmaytak) [inline src/client/Microsoft.Identity.Client.Broker/WamAdapters.cs:180]: nit: curly braces :)

**Review by pmaytak — APPROVED** [review]

File: src/client/Microsoft.Identity.Client.Broker/WamAdapters.cs (hunk 3 above)
Comment (gladjohn) [reply src/client/Microsoft.Identity.Client.Broker/WamAdapters.cs:180]: Yes silent flow using broker (defaultOS account) works as expected

**Review by neha-bhargava — APPROVED** [review]

**Pull request merged**